modules/block/src/main/kotlin/jp/co/soramitsu/iroha2/serde.kt

```diff
@@ -191,18 +218,27 @@ object MintableDeserializer : JsonDeserializer<Mintable>() {
     }
 }
 
+/**
+ * Deserializer for asset definition ID
```

Comment:
Why not to include the type reference just as made for most of other cases? It's possible to have phrasing different from a link content, e.g. [definition ID][DefinitionId]

Reply:
applied your suggestion, thanks!